### DISCUSSION TITLE
Is the Telegram Token encrypted?

### DISCUSSION THREAD

**Discussion by Toby-Mills** [windkh/node-red-contrib-telegrambot#390] [Q&A] [answered]
In the Configuration node UI the token is displayed in plain text. Is it stored in the flows_nodered.json file? Or is it encrypted in the flows_nodered_cred.json file?
I had a look at the flows_nodered.json file, and couldn't find it, so I'm hoping that means it is encrypted.

**Comment by windkh** [comment, marked as answer]
It is and it was always encrypted from the first version on

**Reply by Toby-Mills** [reply]
Great! Thanks for letting me know :)
